### ISSUE TITLE
bug(mat-tab-nav-bar): mat-ink-bar not visible when using <nav mat-tab-nav-bar>

### ISSUE THREAD

**Issue by pwbuys** [angular/components#24018]
### Is this a regression?

- [X] Yes, this behavior used to work in the previous version

### The previous version in which this bug was not present was

12.0.0

### Description

Using the basic example from docs, I'm unable to see the ink bar in the tab label. 
```
<nav mat-tab-nav-bar >
   <a mat-tab-link 
      *ngFor="let link of links"  
      (click)="activeLink = link"
      [active]="activeLink == link"> {{link}} </a>
</nav>
```

Even though it is visible in the [doc example](https://material.angular.io/components/tabs/examples#tab-nav-bar-basic), the issue can still be observed clicking the _open in stackblitz_ button.

### Reproduction

https://stackblitz.com/edit/sg5b1p--run?file=src/app/tab-nav-bar-basic-example.ts

### Expected Behavior

mat-ink-bar should be visible at the bottom of the active tab label

### Actual Behavior

mat-ink-bar is not visible and does not update when the active tab changes. 

### Environment

Angular CLI: 13.0.1
Node: 12.22.1
Package Manager: npm 6.14.12
OS: win32 x64

Angular: 13.0.0
... animations, common, compiler, compiler-cli, core, forms
... platform-browser, platform-browser-dynamic, router

Package                         Version
---------------------------------------------------------
@angular-devkit/architect       0.1300.1
@angular-devkit/build-angular   13.0.1
@angular-devkit/core            13.0.1
@angular-devkit/schematics      13.0.1
@angular/cdk                    13.0.2
@angular/cli                    13.0.1
@angular/material               13.0.2
@schematics/angular             13.0.1
rxjs                            7.4.0
typescript                      4.4.4

**Comment by crisbeto** [comment]
Can you check that you're running Angular core greater than 13.0.1? We had something similar on the docs site and it was due to an issue that was fixed in 13.0.1 https://github.com/angular/material.angular.io/pull/1105.

**Comment by pwbuys** [comment]
I upgraded to 13.0.2 and now it works. Thanks @crisbeto.
